alert-common/src/main/java/com/synopsys/integration/alert/common/rest/ResponseBodyBuilder.java

```diff
@@ -61,12 +68,16 @@ public ResponseBodyBuilder put(String key, String value) {
         return this;
     }
 
-    public ResponseBodyBuilder putErrors(Map<String, String> errors) {
+    public ResponseBodyBuilder putErrors(Map<String, AlertFieldStatus> errors) {
         JsonObject element = new JsonObject();
-        for (Entry<String, String> entry : errors.entrySet()) {
-            element.addProperty(entry.getKey(), entry.getValue());
+        for (Entry<String, AlertFieldStatus> entry : errors.entrySet()) {
+            AlertFieldStatus alertFieldStatus = entry.getValue();
+            JsonObject statusObject = new JsonObject();
+            statusObject.addProperty(PROPERTY_KEY_SEVERITY, alertFieldStatus.getSeverity().name());
```

Comment:
You can add a method to AlertFieldsStatus called getSeverityName that does this
alertFieldStatus.getSeverity().name()